CMakeLists.txt

```diff
@@ -107,17 +107,21 @@ ExternalBackendsInit()
 
 find_package(LLVM 8 CONFIG)
```

Comment:
Could we refactor this code into a separate file under /cmake/modules/ ? 

We don't need to do this now, but after we fix the build it would be good to clean things up.